layout/_macro/post.swig

```diff
@@ -226,17 +226,12 @@
     {%- if not is_index %}
       {{- next_inject('postBodyEnd') }}
 
-      {%- if theme.wechat_subscriber.enable %}
-        {{ partial('_partials/post/wechat-subscriber.swig', {}, {cache: theme.cache.enable}) }}
+      {%- if page.wechat_subscriber.enable %}
```

Comment:
Maybe you can use `post` here?
效果应该是一样的，不过这个文件的上下文统一用的post。后面还有几处。

Reply:
Okay, and I saw https://github.com/theme-next/hexo-theme-next/pull/1232 extract some features to plugins, so I would do it after https://github.com/theme-next/hexo-theme-next/pull/1232 done. 😄

Reply:
I am just worried that swig and nunjucks will behave differently (about global variables) 🤪

Reply:
different? why? They are all implemented through hexo filters and should be the same (about global variables)

Reply:
and in my theme, I'll do same thing like it, it's ejs, it also has same behave.

Reply:
emmm,  If you wanna wechat_subscriber plugin also support defined in front-matter. shoud add a filter.   like:

```
hexo.extend.filter.register('template_locals', locals => {
  const { page, theme } = locals;
  const key = 'wechat_subscriber';
  page[key] = Object.assign({}, theme[key], page[key]);
});
```

Reply:
Because `is_index` will cover up the problem. `post` is `undefined` if `is_index` is `false`.
> and should be the same

Sorry, the answer is no - they are not the same. I know because I have wasted too much time on this issue when migrating to nunjucks. Think about it again, or do some testing.

Reply:
https://github.com/jiangtj/hexo-theme-cake/blob/bdeaeb60a55d5e4129b75d09417f42a45c1246aa/layout/index.ejs#L3
https://github.com/jiangtj/hexo-theme-cake/blob/2719eeced40ee9558005dae36548ab9af480e5d6/layout/index.swig#L4

我也遇到了，post是局部变量，事实上这是框架需要保证的，swig会将局部变量传递，而其他模版似乎需要显性的传递，但无论如何，page变量来自hexo的本地变量，hexo的本地变量在任何模版中都是通用的。

I also encountered, post is a local variable, in fact, this is the framework needs to guarantee, swig will pass local variables, while other templates seem to need explicit delivery, but in any case, the page variable comes from the local variable of hexo, hexo local variables are common in any template.

![image](https://user-images.githubusercontent.com/15902347/67379575-b4a65580-f5bb-11e9-9e33-57027d98e1de.png)

Reply:
是的，确实没有问题，只是为了在`post.swig`中统一写法：要么用`post`，要么用`page`

Reply:
仔细考虑了下，还是觉得使用page更合理，毕竟我是给page赋的值
After careful consideration, I still think it is more reasonable to use page. After all, I assign a value to page.
![image](https://user-images.githubusercontent.com/15902347/67772297-cf356e80-fa94-11e9-8194-a1f57c0dc140.png)